tao_compiler/mlir/disc/tests/tensorflow_ops/floor_div.cc

```diff
@@ -24,6 +24,8 @@ const std::string c_ft_path =
 
 // static shape test case
 TEST(TFFloorDivOpTest, StaticShape2DF32) {
+  // Disable any fast-math setting for this UT.
+  setenv("DISC_CUDA_FAST_MATH_LEVEL", "0", 1);
```

Comment:
nit. You may need `SetUp`/`TearDown` from [here](https://github.com/google/googletest/blob/main/docs/primer.md#writing-the-main-function) instead set and unset env in each test case.

Reply:
I will modify that in the next commit. Thanks.